docs/automated-tests/test-execution/quick-execution.md

```diff
@@ -1,55 +1,57 @@
 ---
-title: How to Execute Test
+title: How to Execute Tests
 page_title: Execute Your First Test in Test Studio
-description: "Test Studio Quick test Execution. Run a Test Studio test. Choose browser to execute the test against."
+description: "Learn how to execute quick tests in Test Studio, modify the quick tests, and access the test results"
 position: 0
 ---
-# How to Execute Test
+# How to Execute Tests
 
-__Quick Test Execution__ in Test Studio is a key feature, which helps in fine tuning the recorded actions, adjust test and step settings, to eventually introduce stable and consistent behavior of the test runs in test list mode, executed on different environments.
+__Quick Test Execution__ helps you fine tune the recorded actions, test, and step settings. It helps you achieve stable and consistent test behavior in test list mode when you execute the test in different environments.
 
-This article will guide you through the execution process in details.
+This article guides you through the quick test execution process in details.
 
 ## Execute a Test
 
-Once you have <a href="/automated-tests/recording/overview" target="_blank">recorded a complete test scenario</a>, the steps, which represents the recorded actions, are listed in the _Test Steps_ pane. These steps can be __executed against any of the supported browsers__, regardless of the browser they were recorded against. To trigger the test run, click the **Execute** button in the ___Test___ ribbon.
+Once you have <a href="/automated-tests/recording/overview" target="_blank">recorded a complete test scenario</a>, the steps, which represents the recorded actions, are listed in the __Test steps__ pane. You can execute these steps against any of the supported browsers, regardless of the browser that you used to record them. 
+
+To trigger the test run, click the **Execute** button in the __Test__ ribbon.
 
 ![Test Studio][1]
 
-In the next dialog, which appears on the screen, __choose the browser to run the test against__. Select any of the browsers listed for execution and click the __Run__ button.
```

Comment:
Minimalism helps users get the needed information faster. In this sentence, you give the same instructions twice: _choose the browser_, and _select any of the browsers_.